### ISSUE TITLE
Crached with pikepdf._qpdf.PdfError

### ISSUE THREAD

**Issue by wpzdm** [jbarlow83/OCRmyPDF#473]
**To Reproduce**
ocrmypdf  --optimize 0  --skip-text --tesseract-timeout=0 \(Annals\ of\ Mathematics\ Studies\ 117\)\ Edward\ Nelson\ -\ Radically\ Elementary\ Probability\ Theory-Princeton\ University\ Press\ \(1987\).pdf \(Annals\ of\ Mathematics\ Studies\ 117\)\ Edward\ Nelson\ -\ Radically\ Elementary\ Probability\ Theory-Princeton\ University\ Press\ \(1987\).o3 

**Example file**
You can get it on libgen: 
http://93.174.95.29/_ads/83F799DACB085903752EE00432FD788C

- [x] This is the input file
- [x] The file contains no personal or confidential information
- [ ] I am the copyright holder for this file
- [ ] I permit this file to be included in the OCRmyPDF test suite under the CC-BY-SA 4.0 license
- [ ] I am not the copyright holder, but this file is available under a free software license

**System:**

- OS: macOS 10.14.5
- OCRmyPDF Version: 9.2.0

**Comment by wpzdm** [comment]
Full console output:
```
Scan: 100%|████████████████████████████████| 111/111 [00:00<00:00, 644.54page/s]
   INFO - Start processing 4 pages concurrent
OCR: 100%|██████████████████████████████| 111.0/111.0 [00:32<00:00,  3.37page/s]
  ERROR - An exception occurred while executing the pipeline
Traceback (most recent call last):
  File "/usr/local/Cellar/ocrmypdf/9.2.0/libexec/lib/python3.7/site-packages/pikepdf/models/metadata.py", line 304, in _load_from
    self._xmp = parse(BytesIO(data))
  File "src/lxml/etree.pyx", line 3469, in lxml.etree.parse
  File "src/lxml/parser.pxi", line 1856, in lxml.etree._parseDocument
  File "src/lxml/parser.pxi", line 1876, in lxml.etree._parseMemoryDocument
  File "src/lxml/parser.pxi", line 1764, in lxml.etree._parseDoc
  File "src/lxml/parser.pxi", line 1127, in lxml.etree._BaseParser._parseDoc
  File "src/lxml/parser.pxi", line 601, in lxml.etree._ParserContext._handleParseResultDoc
  File "src/lxml/parser.pxi", line 711, in lxml.etree._handleParseResult
  File "src/lxml/parser.pxi", line 640, in lxml.etree._raiseParseError
  File "<string>", line 22
lxml.etree.XMLSyntaxError: Start tag expected, '<' not found, line 22, column 20

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/Cellar/ocrmypdf/9.2.0/libexec/lib/python3.7/site-packages/pikepdf/models/metadata.py", line 308, in _load_from
    self._xmp = parse(BytesIO(data))
  File "src/lxml/etree.pyx", line 3469, in lxml.etree.parse
  File "src/lxml/parser.pxi", line 1856, in lxml.etree._parseDocument
  File "src/lxml/parser.pxi", line 1876, in lxml.etree._parseMemoryDocument
  File "src/lxml/parser.pxi", line 1764, in lxml.etree._parseDoc
  File "src/lxml/parser.pxi", line 1127, in lxml.etree._BaseParser._parseDoc
  File "src/lxml/parser.pxi", line 601, in lxml.etree._ParserContext._handleParseResultDoc
  File "src/lxml/parser.pxi", line 711, in lxml.etree._handleParseResult
  File "src/lxml/parser.pxi", line 640, in lxml.etree._raiseParseError
  File "<string>", line 22
lxml.etree.XMLSyntaxError: Start tag expected, '<' not found, line 22, column 20

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/Cellar/ocrmypdf/9.2.0/libexec/lib/python3.7/site-packages/ocrmypdf/_sync.py", line 354, in run_pipeline
    exec_concurrent(context)
  File "/usr/local/Cellar/ocrmypdf/9.2.0/libexec/lib/python3.7/site-packages/ocrmypdf/_sync.py", line 301, in exec_concurrent
    pdf = ocrgraft.finalize()
  File "/usr/local/Cellar/ocrmypdf/9.2.0/libexec/lib/python3.7/site-packages/ocrmypdf/_graft.py", line 282, in finalize
    self.pdf_base.save(self.output_file)
  File "/usr/local/Cellar/ocrmypdf/9.2.0/libexec/lib/python3.7/site-packages/pikepdf/_cpphelpers.py", line 57, in update_xmp_pdfversion
    with pdf.open_metadata(set_pikepdf_as_editor=False, update_docinfo=False) as meta:
  File "/usr/local/Cellar/ocrmypdf/9.2.0/libexec/lib/python3.7/site-packages/pikepdf/models/metadata.py", line 198, in wrapper
    self._load()
  File "/usr/local/Cellar/ocrmypdf/9.2.0/libexec/lib/python3.7/site-packages/pikepdf/models/metadata.py", line 300, in _load
    self._load_from(data)
  File "/usr/local/Cellar/ocrmypdf/9.2.0/libexec/lib/python3.7/site-packages/pikepdf/models/metadata.py", line 310, in _load_from
    raise PdfError() from e
pikepdf._qpdf.PdfError
```

**Comment by jbarlow83** [comment]
Fixed in pikepdf v1.8.3
No change to ocrmypdf

It would probably be easiest to install the latest ocrmypdf and pikepdf in a virtual environment, which you can with `pip install ocrmypdf`. It may be a while before Homebrew picks up the latest pikepdf.
